Documentation/ProjectFile/properties/property/CapillaryPressureVanGenuchten/t_regularization.md

```diff
@@ -0,0 +1,3 @@
+An optional input that indicates whether the capillary pressure is regularized
+ or not.
+Also see \copydoc  MaterialPropertyLib::RegularizedCapillaryVanGenuchten
```

Comment:
copydoc is wrong here. Use \sa  http://www.doxygen.nl/manual/commands.html#cmdsa

Reply:
Changed. Besides, RegularizedCapillaryVanGenuchten is changed to CapillaryPressureRegularizedVanGenuchten.